src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -228,66 +230,54 @@ function MdAutocomplete () {
       }
 
       function getInputElement () {
+        var commonInputAttrs =
+          'type="search" \n' +
+          'name="{{inputName}}" \n' +
+          'autocomplete="off" \n' +
+          'ng-required="$mdAutocompleteCtrl.isRequired" \n' +
+          'ng-disabled="$mdAutocompleteCtrl.isDisabled" \n' +
+          'ng-model="$mdAutocompleteCtrl.scope.searchText" \n' +
+          'ng-readonly="$mdAutocompleteCtrl.isReadonly" \n' +
+          'ng-keydown="$mdAutocompleteCtrl.keydown($event)" \n' +
+          'ng-blur="$mdAutocompleteCtrl.blur()" \n' +
+          'ng-focus="$mdAutocompleteCtrl.focus()" \n' +
+          'ng-class="::inputClass" \n' +
```

Comment:
I'd say so. I added it mainly because the extra menu class did the same, but it would probably be more useful if it was a live binding.